packages/p2p/src/components/orders/orders.jsx

```diff
@@ -1,9 +1,82 @@
 import React from 'react';
-import './orders.scss';
+import { Button, Dialog } from 'deriv-components';
 import { localize } from 'deriv-translations';
+import Popup from './popup.jsx';
+import './orders.scss';
 
 const Orders = () => {
-    return <div>{localize('Orders')}</div>;
+    const [show_popup, setShowPopup] = React.useState(false);
+    const [popup_options, setPopupOptions] = React.useState({});
+
+    const onCancelClick = () => {
+        setShowPopup(false);
+    }
+
+    const onConfirmation = () => {
+        // eslint-disable-next-line no-console
+        console.log('confirmed')
+    }
+
+    const popupSellConfirm = () => {
+        const options = {
+            title: localize('Have you received funds?'),
+            message: localize('Make sure that you have logged in your bank account or other e-wallet to check the receipt.'),
+            need_confirmation: true,
+            offer: {
+                currency       : 'IDR',
+                asset          : 'USD',
+                fix_price      : 12000,
+                amount         : 20,
+            },
+            onClickConfirm: onConfirmation,
+        }
+        setPopupOptions(options);
+        setShowPopup(true);
+    }
+    const popupBuyCancelUnpaid = () => {
+        const options = {
+            title: localize('Confirm this payment?'),
+            message: localize('Make sure you have successfully sent the funds to the seller’s bank account or e-wallet mentioned above.'),
+            has_cancel: true,
+            cancel_text: localize('I didn\'t pay yet'),
+            confirm_text: localize('I\'ve paid'),
+            onClickConfirm: onConfirmation,
+        }
+        setPopupOptions(options);
+        setShowPopup(true);
+    }
+    const popupBuyCancelOrder = () => {
+        const options = {
+            title: localize('Cancel this order?'),
+            message: localize('There will be no refund after canceling the order. If you have paid, please do not cancel the order.'),
+            confirm_text: localize('Cancel this order'),
+            onClickConfirm: onConfirmation,
+        }
+        setPopupOptions(options);
+        setShowPopup(true);
```

Comment:
since all three do the same actions, can we refactor that to one function and pass in the options?